README.md

```diff
@@ -1,12 +1,11 @@
-# Hawk android catcher ![](https://jitpack.io/v/jitpack/maven-simple.svg?style=flat-square)
-### Сборщик ошибок 
-Эта библиотека обеспечивает сбор непроверяемых ошибок во время работы приложения и отправляет их в ваш https://hawk.so личный кабинет.
-Так же существует возможность отправлять отловленные в **try-catch** ошибки
-
+# Hawk Android catcher ![](https://jitpack.io/v/jitpack/maven-simple.svg?style=flat-square)
+### Exception catcher
+This library provides catching errors while the application is running and sending them to your personal account [Hawk](https://www.hawk.so). 
```

Comment:
This library provides in-app errors catching and sending them to the [Hawk.so](https://hawk.so) monitoring system.